app/validation/validator.py

```diff
@@ -333,6 +329,11 @@ def validate_list_collector_type(self, block):
                     errors.append(self._error_message('The block {} contains a list collector without a Radio answer type'
                                                       .format(block['id'])))
 
+                if not self.options_contain_value(answer['options'], add_answer_value):
+                    errors.append(
+                        self._error_message('The block {} contains a list collector with a add_answer_value, that is not in an option'
```

Comment:
The block is a list collector, rather than containing one, so this should be something like "The list collector block {} has an add_answer_value that is not present in the answer values"